Tools/TestWebKitAPI/Tests/WebKitCocoa/WebPushDaemon.mm

```diff
@@ -1142,6 +1046,17 @@ HTTPServer server({
 
 #if ENABLE(INSTALL_COORDINATION_BUNDLES)
 #if USE(APPLE_INTERNAL_SDK)
+static void showNotificationsViaWebPushDInsteadOfUIProcess(WKWebViewConfiguration *configuration)
```

Comment:
I renamed `enableNotifications` to `showNotificationsViaWebPushDInsteadOfUIProcess` since that's what these preferences actually enable.